src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -222,6 +222,7 @@ def __call__(
         offset = self.scheduler.config.get("steps_offset", 0)
         init_timestep = int(num_inference_steps * strength) + offset
         init_timestep = min(init_timestep, num_inference_steps)
+        # FIXME
         if isinstance(self.scheduler, LMSDiscreteScheduler):
             timesteps = torch.tensor(
                 [num_inference_steps - init_timestep] * batch_size, dtype=torch.long, device=self.device
```

Comment:
The timesteps in LMS are in reverse: ideally need to keep them in a consistent order between schedulers